src/components/React/ColorMappingController.js

```diff
@@ -76,91 +34,98 @@ export default connect(mapStateToProps, {
       super(props);
 
       this.state = {
-        colorMapSelected: "",
+        colorMap: colorMaps[0],
         colorMapModal: false,
-        currentMapColor: "./colormaps/viridis.png",
       };
 
       this.showModal = this.showModal.bind(this);
```

Comment:
I think a drop down is better than modal+data table, what is the hesitation on simplifying/deleting the code to just that?

Reply:
Wanted confirmation of the design change, will delete!